application/hart0/hss_state_machine.c

```diff
@@ -107,7 +107,7 @@ void RunStateMachine(struct StateMachine *const pCurrentMachine)
                 prevState =    pCurrentMachine->prevState;
                 currentState = pCurrentMachine->state;
 
-#ifdef DEBUG
+#if 0
```

Comment:
*quesiton* Is `DEBUG` defined because of the `CONFIG_DEBUG_LOG_STATE_TRANSITIONS=y` within `bni-delta_defconfig` [here](https://github.com/ButterflyNetwork/hart-software-services/pull/1/files#diff-0bdde9de2c8838231af241b4885b648a1a1c9d5bca9936eec3d4401bc8467b45R140)? Wouldn't we want to set that flag to `No` if that's the case versus modifying this?

Perhaps there are other useful things we get by having `CONFIG_DEBUG_LOG_STATE_TRANSITIONS=y`?

Reply:
Reverted back to DEBUG so that all debug messages are controlled by configuration option(s) in the appropriate defconfig